src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -629,4 +629,22 @@ public function getQueueableConnection()
 
         return $connection;
     }
+
+    /**
+     * Get the Eloquent query builder from the collection.
+     *
+     * @return Illuminate\Database\Eloquen\Builder
+     *
+     * @throws \LogicException
+     */
+    public function toQuery()
+    {
+        $model = $this->first();
+
+        if (! $model) {
+            throw new LogicException('can not get Eloquent QueryBuilder from an empty Collection.');
+        }
+
+        return $model->newModelQuery()->whereKey($this->modelKeys());
```

Comment:
I assume you wanted "where in" and not "where". Have you tried this code for real?

Reply:
<img width="628" alt="Screen Shot 2020-06-27 at 1 04 03 AM" src="https://user-images.githubusercontent.com/29691074/85907526-1df8fb00-b812-11ea-9d98-487a59d24c4d.png">

yes I tried it and **loadCount** function is implemented the same way